cypress/integration/Internationalization.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Internationalization
+  As a user who is not very fluent in English
+  I would like to see the user interface translated to my preferred language
+  In order to be able to understand the interface
+
+  Background:
+    When I visit the "login" page
```

Comment:
```suggestion
    Given I am on the "login" page
```
Don't mess up `Given`, `When`, `Then` order if possible.

Reply:
It does not worked with variables. And also did I not want to dublicate any step logic as it does exactly the same.

Reply:
It is absolutey :ok: to have duplicated step definitions. You should write your features such that they read nicely, it's not required to share step definitions to save code. With `cucumber-js` I use the step definition generator.

What did not work with variables?

You have to use `{string}` to match anything in double quotation marks.